setup.py

```diff
@@ -32,117 +32,115 @@
 readme = open('README.rst').read()
 
 install_requires = [
-    'amqp>=1.4.9,<2.0',
-    'celery<4.0',
-    'Flask-Gravatar>=0.4.2',
-    'HarvestingKit>=0.6.2',
-    'plotextractor>=0.1.2',
-    'refextract>=0.1.0',
-    'Sickle>=0.5.0',
-    'orcid',
-    'raven<=5.1.0',
-    'retrying',
-    'flower',
-    'rt',
-    'langdetect>=1.0.6',
-    'librabbitmq>=1.6.1',
-    'idutils>=0.2.1',
-    'invenio-access>=1.0.0a7',
-    'invenio-accounts>=1.0.0a12',
-    'invenio-admin>=1.0.0a3',
-    'invenio-assets>=1.0.0b2',
-    'invenio-base>=1.0.0a11',
-    'invenio-celery>=1.0.0a4',
-    'invenio-classifier~=1.1.0',
-    'invenio-collections>=1.0.0a3',
-    'invenio-config>=1.0.0a1',
-    'invenio-i18n>=1.0.0a4',
-    'invenio-indexer>=1.0.0a7',
-    'invenio-jsonschemas>=1.0.0a4',
-    'invenio-logging>=1.0.0a3',
-    'invenio-mail>=1.0.0a4',
-    'invenio-oauthclient>=1.0.0a8',
-    'invenio-orcid>=1.0.0a1',
-    'invenio-records>=1.0.0a16',  # Add [versioning] in the future
-    'invenio-rest[cors]>=1.0.0a7',
-    'invenio-search>=1.0.0a7',
-    'invenio-records-rest>=1.0.0a17',
-    'invenio-records-ui>=1.0.0a6',
-    'invenio-files-rest>=1.0.0a3',
-    'invenio-records-files>=1.0.0a5',
-    'invenio-userprofiles>=1.0.0a7',
-    'invenio-oaiharvester==1.0.0a2',
-    'invenio-utils==0.2.0',  # Not fully Invenio 3 ready
-    'invenio>=3.0.0a1,<3.1.0',
-    'inspire-crawler~=0.2.7',
-    'inspire-schemas~=31.0,>=31.1.0',
-    'dojson>=1.3.0',
-    'Flask>=0.11.1',
-    'Flask-Breadcrumbs>=0.3.0',
-    'Flask-Caching>=1.0.1',
-    'Flask-Script>=2.0.5',
-    'flask-shell-ipython>=0.2.2',
-    'fs<2.0',  # TODO: remove once invenio-files-rest#130 is fixed
-    'jsmin',
-    'pytest-runner>=2.7.0',
+    'amqp>=1.4.9,~=1.4',
```

Comment:
Oh, and I don't believe that a constraint like `~=1.4` actually picks up version `1.5`, for example. I think you have to write `~=1.0`.